manifests/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -248,10 +248,14 @@ spec:
 {{- else }}
 {{ toYaml .Values.global.defaultResources | trim | indent 12 }}
 {{- end }}
+          {{- if and .Values.global.controlPlaneSecurityEnabled .Values.global.SecretVolumeMtlsCerts }}
+          # Use the key and cert mounted to /etc/certs/ for the in-cluster mTLS communications.
+          # TODO: Pilot needs to read from the local file when using the mounted key and cert.
```

Comment:
not sure what this means?

Reply:
Yes, AFAIK, Pilot now can use the Citadel module to provision its cert directly, or use Chiron to provision its cert from apiserver. It can't directly load its cert from files, right (maybe I'm wrong)?